fluidBuild.config.cjs

```diff
@@ -3,6 +3,8 @@
  * Licensed under the MIT License.
  */
 
+// ts-check
```

Comment:
What is this doing exactly? Should it be commented?

Reply:
It [enables type-checking on the file](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html#ts-check), but I had it wrong too. Fixed in latest with an additional comment.